lua/lazy/core/plugin.lua

```diff
@@ -110,7 +110,11 @@ function Spec:add(plugin, results)
     plugin.dev
     and (not Config.options.dev.fallback or vim.fn.isdirectory(Config.options.dev.path .. "/" .. plugin.name) == 1)
```

Comment:
Will throw an error trying to concatenate a function on this line if fallback and dev are true